pkg/generators/openapi.go

```diff
@@ -452,26 +433,29 @@ func (g openAPITypeWriter) generateExtensions(CommentLines []string) error {
 			extensions = append(extensions, NameValue{parts[0], parts[1]})
 		}
 	}
-	// Next, generate extensions from "tags".
-	for _, tagKey := range sortedMapKeys(tagToExtension) {
-		tagValue, err := getSingleTagsValue(CommentLines, tagKey)
-		if err != nil {
-			return err
-		}
-		if len(tagValue) > 0 {
-			extensions = append(extensions, NameValue{tagToExtension[tagKey], tagValue})
-		}
-	}
-	// If there are generated extensions, write them out to schema.
-	// Example: "x-kubernetes-patch-strategy" : "merge"
-	if len(extensions) == 0 {
+	// Next, generate "tag" extensions (e.g. "+patchMergeStrategy", "+listAttribute=...")
+	tagExtensions := getTagExtensions(CommentLines)
+	if len(extensions) == 0 && len(tagExtensions) == 0 {
 		return nil
 	}
+	// If any extensions exist, then emit code to create them.
 	g.Do("VendorExtensible: spec.VendorExtensible{\nExtensions: spec.Extensions{\n", nil)
 	for _, extension := range extensions {
 		g.Do("\"$.$\": ", extension.Name)
 		g.Do("\"$.$\",\n", extension.Value)
 	}
+	for _, tagExtension := range tagExtensions {
```

Comment:
Please add tests until you find the bug in this code. :)

Reply:
I found one. I don't know if it's the same, but I found one :-)